### ISSUE TITLE
Screen reader doesn't announce color value changes.

### ISSUE THREAD

**Issue by azaslonov** [Simonwep/pickr#271]
#### What is the current behavior?
Whenever you change the color, the screen reader should announce the new color value. I believe this could be solved with "live regions" feature: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

**Comment by Simonwep** [comment]
Duplicate of #11. Please add a comment there and don't open another conversation :)

**Comment by azaslonov** [comment]
Thanks!
